test/framework/contracts.ts

```diff
@@ -18,7 +18,6 @@ export async function deployContract<T extends DeployableContract<T>>(
 ): Promise<T> {
     const factory = await ethers.getContractFactory(name)
     const contract = <T>(<unknown>await factory.deploy(...args))
-
```

Comment:
question: did you delete the whitespace, or was that part of your IDE formatting?

Reply:
Think I was initially using this code to deploy the contract. Reset to the `main` copy.